finance-api/overview.md

```diff
@@ -0,0 +1,19 @@
+Finance API overview
+===
+The Finance API fetches information about transactions that are made through Zettle. For example, card payments made with a Zettle card reader.
+
+# Understand how card payments work at Zettle
```

Comment:
I don't think that "how card payments work at Zettle" is relevant in the finance context; for the purposes of the finance API, it seems to me that people need to understand what the accounts mean (preliminary, liquid) and perhaps where the money on the preliminary/liquid account could come from--card present payments, payment links, QRC payments... depending on the market, of course. 

So I would propose removing "How card payments work" from the Finance API docs entirely. 

What do you think @Astro-Choco?